source/guides/non-denizen/resource-packs.md

```diff
@@ -0,0 +1,248 @@
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+### Inside The Root Directory
+Directory: `.minecraft\resourcepacks\MyResourcePack\`
+
+The main directory within your resource pack folder should contain both:
+
+`Assets` Folder - This is where all your files are placed.
+`pack.mcmeta` - This is how Minecraft knows what format your Resource Pack is.
+Optimally, you can also include:
+
+* pack.png - This is a `64x64` custom image for your pack!
+
+#### Example File: `pack.meta`
+Inside the `pack.meta` file, you will find this is formatted in a `json` format. You need two named strings: `pack_format` and `description`.
+Here is what it looks like inside:
+```json
+{
+   "pack": {
+      "pack_format": 5,
+      "description": "My Fancy Resource Pack"
+   }
+}
+```
+
+##### pack.meta File Key: `pack_format`
+This is the indicator to Minecraft what version this pack is.
+1 indicates versions `1.6` - `1.8`.
+2 indicates versions `1.9` - `1.10`,
+3 indicates versions `1.11` - `1.12`,
+4 indicates versions `1.13` - `1.14`,
+5 indicates versions `1.15` - `1.16`.
+
+##### pack.meta File Key: `description`
+This can be blank, or you can optimally fill this with something fancy. Unicode characters must be written pre-escaped, like this: `\uCODE`; two examples being: `\u2588` for `█`, and `\u00A7` for `§`, the section sign symbol which parses [valid color tags](https://minecraft.gamepedia.com/Formatting_codes) you use to parse colors in minecraft chat. If you want red text, your text will look something like `\u00A74Dark Red!`.
+*Note: Color before formatting; Formatting codes persist after a color code, Not vise-versa!*
+
+You can find special characters in your Character Map. If you're on a Windows operating system, `Start` > `Windows Accessories`. If you're in Linux using GNOME and Unity, `Gucharmap character map` is a part of `GNOME desktop`. If you run a Gnome desktop - you can access it in any of these following ways:
+- Menu on the top of the screen, `(language)` > `Character Map`.
+- `Gucharmap` in terminal.
+- `Applications` > `Accessories` > `Character Map`.
+
+You can also google search for unicode characters.
+
+### Inside The Assets Folder Directory
+Directory: `.minecraft\resourcepacks\MyResourcePack\assets\`
+This directory should be empty except for the one folder directory: `minecraft`
+toss it in and leave everything else out of here.
+
+### Inside The Minecraft Folder Directory
+Directory: `.minecraft\resourcepacks\MyResourcePack\assets\minecraft\`
+
+Depending on what content you plan on changing, you need to opt creating these folders:
```

Comment:
This phrasing seems a bit off. Maybe something more like `Depending on what content you plan on changing, you can create any of the following folders:`